tests/test_vta.py

```diff
@@ -0,0 +1,73 @@
+"""PyNE vta module tests"""
+import os
+
+import nose
+
+from nose.tools import assert_equal, assert_not_equal, assert_raises, raises, \
+                       assert_in, assert_true
+
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+
+import numpy as np
+from pyne import vta
+
+
+def test_ray_voxel_traverse_2D():
+    x_bound = [0, 1, 2, 3, 4, 5, 6]
+    y_bound = [0, 1, 2, 3, 4, 5, 6]
+    z_bound = [0, 1]
+    bounds = [x_bound, y_bound, z_bound]
+    A = np.array([3, 5.5, 0.5])
```

Comment:
I'd recommend changing `A`/`B` to `ray_start`/`ray_end`. In other places A, B, C are used to represent triangle vertices.